src/admin.c

```diff
@@ -623,13 +623,13 @@ static bool admin_show_users(PgSocket *admin, const char *arg)
 	return true;
 }
 
-#define SKF_STD "ssssssisiTTiiississi"
-#define SKF_DBG "ssssssisiTTiiississiiiiiiii"
+#define SKF_STD "issssssisiTTiiississi"
+#define SKF_DBG "issssssisiTTiiississiiiiiiii"
 
 static void socket_header(PktBuf *buf, bool debug)
 {
 	pktbuf_write_RowDescription(buf, debug ? SKF_DBG : SKF_STD,
-				    "type", "user", "database", "replication", "state",
+				    "id", "type", "user", "database", "replication", "state",
```

Comment:
As I mentioned above, move this field to the end.

Reply:
Moved right after `prepared_statements` and before `recv_pos`